### ISSUE TITLE
Tensor's host is null.

### ISSUE THREAD

**Issue by GHLab** [alibaba/MNN#603]
Hi,
I'd like to run https://github.com/OverEuro/deep-head-pose-lite so I converted it to onnx by below code.

model = stable_hopenetlite.shufflenet_v2_x1_0()
saved_state_dict = torch.load('../model/shuff_epoch_120.pkl', map_location="cpu")
model.load_state_dict(saved_state_dict, strict=False)
model.eval()

x = torch.rand(1, 3, 224, 224)
torch_out = model(x)

torch_out = torch.onnx._export(model, x, "hopenet_lite.onnx", export_params=True)

And converted the onnx to mnn and I tested it by below link.
https://github.com/GHLab/deep-head-pose-lite-mnn/blob/master/HeadPoseDetectorMNN.cpp

But the tensor result is null.
Do you have any guesses to resolve this problem?
I checked the onnx has no problem by demo and ncnn project using same onnx shows correct result.
You can run it by https://github.com/GHLab/deep-head-pose-lite-onnx and can check https://github.com/GHLab/deep-head-pose-lite-ncnn

**Comment by jxt1234** [comment]
Please update code and recompile the mnnconverter and then re-convert your model

**Comment by GHLab** [comment]
@jxt1234 Really thank you! I resolved the issue.

**Comment by JimKimHome** [comment]
> @jxt1234 Really thank you! I resolved the issue.

Hello，I have the same problem as yours, I convert shufflet_v2_x0.5_ssd.pth to .onnx model to .mnn model, and when I load it in andriod studio and run the inference session,I got no data from output tensor, I wander how you fix this problem ,could you tell me how to fix it?
[shufflenet_v2_ssdlite_w_h.zip](https://github.com/alibaba/MNN/files/5229229/shufflenet_v2_ssdlite_w_h.zip)
